source/topics/account/how-to-invite-training-user.md

```diff
@@ -0,0 +1,13 @@
+To invite new team members to training: 
+
+1) Create a _Training-only_ Role (if it does not yet exist)
```

Comment:
This is not required. Please explain when and why a role like this is necessary.